android/modules/ui/src/java/ti/modules/titanium/ui/WebViewProxy.java

```diff
@@ -130,6 +130,15 @@ public Object evalJS(String code, @Kroll.argument(optional = true) KrollFunction
 		return view.getJSValue(code);
 	}
 
+	@Kroll.method
+	public void injectCode()
```

Comment:
Should this method be called `injectEventBindings()`?

```suggestion
	public void injectEventBindings()
```